lib/mix/tasks/vbt/bootstrap.ex

```diff
@@ -11,5 +11,47 @@ defmodule Mix.Tasks.Vbt.Bootstrap do
       ~w/makefile docker circleci heroku github_pr_template credo dialyzer/,
       &Mix.Task.run("vbt.gen.#{&1}", args)
     )
+
+    Mix.shell().info(manual_instructions())
+  end
+
+  defp manual_instructions do
+    """
+    You also need to make the following changes in your mix.exs file:
+
+        def project do
+          [
+            # ...
+            preferred_cli_env: preferred_cli_env(),
+            dialyzer: dialyzer()
+          ]
+        end
+
+        def deps do
+          [
+            # ...
+            {:dialyxir, "~> 0.5", runtime: false}
+          ]
+        end
+
+        defp aliases do
+          [
+            # ...
+            test: ["ecto.create --quiet", "ecto.migrate", "test"],
+            credo: ~w/compile credo/
+          ]
+        end
+
+        defp preferred_cli_env() do
+          [credo: :test, dialyzer: :test]
+        end
+
+        defp dialyzer() do
+          [
+            plt_add_apps: [:ex_unit],
```

Comment:
One more thing, without `:mix` in `plt_add_apps` I get the following error: `:0: Unknown function 'Elixir.Mix.Project':config/0`

Should this be added, or am I doing something else wrong?

Reply:
This one is a bit tricky. The problem is that mix function shouldn't be used in the production code. OTOH, mix tasks are allowed to use `mix`, because they are not meant to be used in production.

A proper solution would be to run dialyzer in all three mix envs (dev, test, prod), conditionally compile mix tasks only in dev, and have conditional definition of `plt_add_apps` (or use `extra_applications` instead). 

However, this feels like an overkill, and it will affect the CI build duration, so I think we should just be pragmatic and add `:mix` in this list. I'll make the change here, and in parallel we could try the proper solution in some smaller project.